news-catalyst/next-tinynewsdemo
# Support for previewing articles

Issue #47 

This PR adds support for previewing articles, which is part of the work the linked issue above requires.

Now I need to figure out how this would work with regards to the data in webiny... I'll put my thoughts on the remaining to-dos in issue #47 

How to test this:

* make sure you add a token to your .env.local: `PREVIEW_TOKEN="SOMETHINGSECRET_GOES_HERE"`
* open the url: http://localhost:3000/api/preview?secret=SOMETHINGSECRET_GOES_HERE&slug=heres-how-philadelphia-public-schools-will-operate-this-fall

You should be redirected to the correct article page. Next will have set some preview cookies.
